packages/nuekit/src/deps.js

```diff
@@ -1,8 +1,13 @@
 
-import { join, normalize, dirname, extname, basename, sep } from 'node:path'
+import { join, dirname, extname, basename, sep } from 'node:path'
+
+// Normalize path separators to POSIX format
+function toPosix(path) {
+  return path.split(sep).join('/')
+}
```

Comment:
maybe move this function to a proper file, export it, and import it where needed, and reuse it in the places where you do `.replace(/\\/g, '/')`